pkg/kube/kube_nodes.go

```diff
@@ -86,7 +87,9 @@ func FetchNamespaceNodeInfo(ctx context.Context, kube *corev1client.CoreV1Client
 
 	for i := 0; i < len(pods.Items); i++ {
 		res := <-walletInfoResponseC
-		if res.Error == nil {
+		if res.Error != nil {
+			log.Printf("wallet info error: %s\n", res.Error.Error())
+		} else {
```

Comment:
@gacevicljubisa out of curiosity, why was this logging needed?

node funder tries to get wallet info from all pods, and when requests succeeds it means that pod is actually bee node. this was the reason why this error was not logged 1) to avoid clutter of logs 2) to not alert user with this error because it was expected here.